core/unit_test/TestViewAPI_e.hpp

```diff
@@ -273,7 +273,11 @@ struct TestViewAllocationLargeRank {
   ViewType v;
 };
 
+// Orin and other Jetson devices come with smaller memory capacity, 8GB total
 TEST(TEST_CATEGORY, view_allocation_large_rank) {
+  #ifdef KOKKOS_ARCH_AMPERE87
+    GTEST_SKIP() << "skipping for Jetson devices that have only 8GB memory";
```

Comment:
```suggestion
    GTEST_SKIP() << "skipping for Jetson devices since they only have 8GB memory";
```

Reply:
Ignoring suggestion since I am editing on GH to fix the format and the language fix is not critical and we might have to re-apply clang-format.